tests/pytorch/test_fusible_ops.py

```diff
@@ -24,10 +26,20 @@
     ForwardLinearBiasAdd,
 )
 from transformer_engine.pytorch.tensor import QuantizedTensor
-from transformer_engine.pytorch.tensor.float8_tensor import Float8Tensor, Float8Quantizer
+from transformer_engine.pytorch.tensor.float8_tensor import (
+    Float8Tensor,
+    Float8CurrentScalingQuantizer,
+    Float8Quantizer,
+)
+from transformer_engine.pytorch.tensor.mxfp8_tensor import MXFP8Tensor
 from transformer_engine.pytorch.utils import is_bf16_compatible
 import transformer_engine_torch as tex
 
+# Import utility functions
+_current_file = pathlib.Path(__file__).resolve()
+sys.path.append(str(_current_file.parent))
```

Comment:
Not a fan of this. But I see we currently do this for other tests as well so for this PR it's ok. In a separate refactor we should convert the `tests` dir to also be a module.

Reply:
I agree. Ideally we would use relative imports like `from .utils import dtype_tols`, but Python doesn't allow that if you are running the script directly, which is how Pytest runs the tests.